webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import CommonForm from './CommonForm';
+
+const Select = (props) => {
+  return (
+    <CommonForm label={props.label}
+    className={`common-select ${props.className}`}>
+      <select className="form-control"
+              value={props.value}
+              onChange={props.onChange}
+      >
+        <option value="">{__('Please select')}</option>
```

Comment:
this is hidden when a user selects another options, it is regular select behavior, same as other options in the select